weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTableButtons.tsx

```diff
@@ -0,0 +1,315 @@
+import {Box, Popover} from '@mui/material';
+import {GridFilterModel, GridSortModel} from '@mui/x-data-grid-pro';
+import {Button} from '@wandb/weave/components/Button';
+import {Checkbox} from '@wandb/weave/components/Checkbox/Checkbox';
+import {TextField} from '@wandb/weave/components/Form/TextField';
+import {Icon, IconName} from '@wandb/weave/components/Icon';
+import {IconOnlyPill} from '@wandb/weave/components/Tag';
+import {Tailwind} from '@wandb/weave/components/Tailwind';
+import {maybePluralize} from '@wandb/weave/core/util/string';
+import React, {Dispatch, FC, SetStateAction, useRef, useState} from 'react';
+
+import {useWFHooks} from '../wfReactInterface/context';
+import {
+  ContentType,
+  fileExtensions,
+} from '../wfReactInterface/traceServerClientTypes';
+import {WFHighLevelCallFilter} from './callsTableFilter';
+import {useFilterSortby} from './callsTableQuery';
+
+const MAX_EXPORT = 10_000;
+
+type SelectionState = {
+  allChecked?: boolean;
```

Comment:
I'm wondering why these fields are optional. They seem exclusive so maybe an enum string would be better?